lib/extensions.js

```diff
@@ -0,0 +1,75 @@
+// Imports
+const fs = require('fs')
+const path = require('path')
+const foundationExtensions = require('../app/config').foundationExtensions || ['govuk-frontend']
+
+// Utils
+const pathFromRoot = (...all) => path.join.apply(null, [__dirname, '..'].concat(all))
+const pathToHookFile = packageName => pathFromRoot('node_modules', packageName, 'govuk-prototype-kit.config.json')
```

Comment:
In this instance `package` is used to refer to the `module` which may or may not be an `extension`.  Package seems like the wrong word but `extension` seems to be preemptive because this path will be used to test whether the module is an extension or not.

Reply:
> path will be used to test whether the module is an extension or not

seems like a good reason to use `package` 👍